plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -342,3 +335,16 @@ def parse_commandline_args(
                 )
 
         return parameters
+
+
+def chunks(
+    batch_of_trials: BatchOfTrialType, n: Optional[int]
+) -> Iterable[BatchOfTrialType]:
+    """
+    Chunk the batch into chunks of upto to n items (each)
+    """
+    print("chunk")
```

Comment:
print?

Reply:
1. make static.
2. add tests for this function.

Reply:
Fixed